app/Auth0/Repositories/EloquentAuth0ClientRepository.php

```diff
@@ -46,6 +46,19 @@ public function getByIntegrationId(UuidInterface $integrationId): array
             ->toArray();
     }
 
+    public function getById(UuidInterface $id): ?Auth0Client
+    {
+        $auth0ClientModel = Auth0ClientModel::query()
```

Comment:
I would use a `findOrFail` here. Because calling this code with a wrong `id` is something that needs fixing.